CUJExecutor/app/src/androidTest/java/com/example/firstapp/ShellUtilityTest.java

```diff
@@ -38,31 +40,26 @@
  */
 public class ShellUtilityTest {
     private static final String BASIC_SAMPLE_PACKAGE = "com.example.firstapp";
+    private static final String MAPS_PACKAGE = "com.google.android.apps.maps";
     private static final int TIMEOUT = 6000;
     private static final String entered_text = "Not bad";
     private ShellUtility shellUtility;
 
 
     @Before
-    public void openBasicApp() throws IOException, InterruptedException {
+    public void getShellUtility() throws IOException, InterruptedException, RemoteException, UiObjectNotFoundException {
         shellUtility = new ShellUtility(TIMEOUT);
-        shellUtility.forceQuitApp(BASIC_SAMPLE_PACKAGE);
-        shellUtility.launchApp(BASIC_SAMPLE_PACKAGE);
     }
 
     /**
      * LAUNCHING AND QUITING
      */
-    @Test
-    public void launching() {
-        Boolean appeared = shellUtility.device.wait(Until.hasObject(By.pkg(BASIC_SAMPLE_PACKAGE).depth(0)), TIMEOUT);
-        assertEquals(true, appeared);
-    }
 
 
 
     @Test
-    public void quitingApp() throws IOException, InterruptedException {
+    public void quitingApp() throws IOException, InterruptedException, RemoteException, UiObjectNotFoundException {
+        shellUtility.launchApp(MAPS_PACKAGE);
```

Comment:
Your unit tests should not depend on variables that are not under your control, like assuming that there will be Google Maps on the phone.